src/data.jl

```diff
@@ -0,0 +1,41 @@
+function register_datadep()
```

Comment:
Its weird for an imputation library to come with its own data.
But I guess its Ok.
It does raise a lot of questions for semver though, if you ever want ot change the data.

Reply:
This is a cute little data library though.
I like these kinds of wrappers around DataDeps a lot